pyfolio/timeseries.py

```diff
@@ -169,36 +129,23 @@ def calmar_ratio(returns, period=DAILY):
     ----------
     returns : pd.Series
         Daily returns of the strategy, noncumulative.
-         - See full explanation in tears.create_full_tear_sheet.
+        - See full explanation in :func:`~pyfolio.timseries.cum_returns`.
```

Comment:
several `timseries` -> `timeseries`, might be worth a find and replace :trollface: